Standards/CCSMB-12.md

```diff
@@ -0,0 +1,90 @@
+# *CCSMB 12:* 3D JSON Array Format
+
+*Author: Dimaguy*
+
+*Version: 1.0.0*
+
+*Last revised: 2024-04-02*
+
+The words MUST, MUST NOT, SHOULD, SHOULD NOT, and MAY are defined in [RFC 2119](https://www.rfc-editor.org/rfc/rfc2119).
+This RFC introduces the 3D JSON Array Format, used for printing 3D Blocks in ComputerCraft using the mod [sc-peripherals](https://github.com/SwitchCraftCC/sc-peripherals)
+
+## Quick information
+
+| Information |                           |
+| ----------- | ------------------------- |
+| Version     | 1.0.0                     |
+| Type        | Model Array file format   |
+| MIME        | `model/3dja`              |
+| Extensions  | `.3dja`                   |
+
+## Technical details
+
+The file MUST be composed of a single JSON object ([RFC 8259](https://datatracker.ietf.org/doc/html/rfc8259), "The JavaScript Object Notation (JSON) Data Interchange Format") with the following ruleset:
+- The writer SHOULD specify global non-default (according to 3dj) common settings for the printed 3D blocks in the global namespace of the JSON object.
+- The writer MAY define a dictionary of presets (which are tint/texture combinations), if this isn't done every block MUST use their own texture and tint for each shape.
+- The writer MUST define an array of block models, following the specifications of the [3dj](https://docs.sc3.io/features/sc-peripherals.html#_3dj-format) format with the addition of a coordinates array (if the user foregoes presets each block model object should be compatible with a 3dj file, the coordinates field SHOULD be disregarded by 3dj-only readers).
```

Comment:
See https://github.com/CCSMB/Standards/pull/30#discussion_r1563100765. Also consider referencing the CCSMB specification if it exists.

Reply:
There's no CCSMB merged in for 3dj just yet, and citing a PR doesnt sound very solid, I think that's a policy that needs to be worked out amongst ccsmb members

Reply:
JSON was referred to in a slightly different way than 3dj that justified a footnote where 3dj doesn't. I'm marking this conversation as solved now